rts/lib/headlessStubs/sdlstub.c

```diff
@@ -363,6 +363,23 @@ extern DECLSPEC void SDLCALL SDL_StartTextInput(void) {
 extern DECLSPEC void SDLCALL SDL_StopTextInput(void) {
 }
 
+extern DECLSPEC int SDLCALL SDL_CaptureMouse(SDL_bool capture) {
```

Comment:
maybe could be a standalone change? idk

Reply:
I believe this PR contains the first code that actually needs those extern stubs

Reply:
alright, but i think it would be decent to get them out of the way regardless